modules/reference/pages/properties/object-storage-properties.adoc

```diff
@@ -402,7 +402,7 @@ NOTE: The container must belong to <<cloud_storage_azure_storage_account>>.
 
 === cloud_storage_azure_hierarchical_namespace_enabled
 
-Whether or not an Azure hierarchical namespace is enabled on the <<cloud_storage_azure_storage_account>>. If this property is not set, <<cloud_storage_azure_shared_key>> must be set, and each node tries to determine at startup if a hierarchical namespace is enabled. Setting this property to `true` disables the check and treats a hierarchical namespace as active. Setting to `false` disables the check and treats a hierarchical namespace as not active.
+Whether or not an Azure hierarchical namespace is enabled on the <<cloud_storage_azure_storage_account,`cloud_storage_azure_storage_account`>>. If this property is not set, <<cloud_storage_azure_shared_key,`cloud_storage_azure_shared_key`>> must be set, and each node tries to determine at startup if a hierarchical namespace is enabled. Setting this property to `true` disables the check and treats a hierarchical namespace as active. Setting to `false` disables the check and treats a hierarchical namespace as not active.
```

Comment:
this description was recently updated in redpanda source code
https://github.com/redpanda-data/redpanda/blob/42efc2b0efd008a2fef88ff4712dbe42b5f85ea6/src/v/config/configuration.cc#L2222-L2232
(functionally is the same, it's just a rewording)

Reply:
ack. I believe this hasn't landed on a version release yet, that's why we don't have the change. This delta here is just about the formatting